src/components/index.tsx

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components';
 export const ContentContainer = styled.div`
   display: block;
   padding: 24px;
+  padding-top: 12px;
```

Comment:
Is ContentContainer used in other places? Did you check that this doesn't mess them up?

Reply:
Yeah this won't break anything, the Figma is a bit ambivalent about navbar/content spacing anyway and this will keep everything above the fold more or less. Shifting content up 12px shouldn't be too dramatic of a change.